docs-sources/includes/_Requirements.md

```diff
@@ -2,19 +2,20 @@
 
 ## Node.js
 
-<aside class="notice">
```

Comment:
I had to remove the aside, since it seems that it doesn't support links in its content.

Reply:
Surely `<a href="...">...</a>` should work?

Reply:
Also, where would you need a link?

Reply:
Link to the LTS schedule

Reply:
Do you want me to put the notice back? With the added information, I'm not sure if you still prefer that there or not.

Reply:
I think that would be preferable.